### DISCUSSION TITLE
Sync to another user

### DISCUSSION THREAD

**Discussion by thedinz** [Taxel/PlexTraktSync#1072] [Q&A]
I guess almost as if i set it up from scratch i want to change the user that syncs, or use both (thats simple i just change that in the config file) but what i want to do that i cant figure out is i have been syncing to trakt as user 2, i want to now sync from trakt back to user 1

This way going forward user 1 and 2 will be in sync.

How do i do that? What commands must i run to get it to sync from trakt back to my user of choice?

Lastly, is it syncing to trakt either 1 user or all? Or can i specifiy which plex accounts to sync, right now its true or false.

**Comment by thedinz** [comment]
Nevermind got it.

**Reply by glensc** [reply]
you forgot to close and mark as answer. more more specifically explain others the answer, as described in the guide
